source/MaterialXView/Main.cpp

```diff
@@ -14,6 +14,8 @@ const std::string options =
 "    --mesh [FILENAME]              Specify the filename of the OBJ mesh to be displayed in the viewer\n"
 "    --meshRotation [VECTOR3]       Specify the rotation of the displayed mesh as three comma-separated floats, representing rotations in degrees about the X, Y, and Z axes (defaults to 0,0,0)\n"
 "    --meshScale [FLOAT]            Specify the uniform scale of the displayed mesh\n"
+"    --enableTurntable[BOOLEAN]     Specify whether to enable turntable rendering of the scene\n"
+"    --turntableAmount [FLOAT]      Specify the amount to incrementatlly rotate the scene when turntable rendering is enabled (defaults to 1 degree)\n"
```

Comment:
This might be better stated as "--turntableSteps", where the current rotation is computed each frame from the current and total step counts.  This would avoid the potential for floating-point precision issues when incrementing rotation angles on a per-frame basis.

Reply:
Good suggestion. Updated now. Thanks.